### PR TITLE
Vagrant updates

### PR DESCRIPTION
**Pull request by chris-moreton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -144,9 +144,9 @@ script:
 #-----------------------------------------------------------------------------
 
 after_script:
-  - if [[ $TRAVIS_BRANCH == 'master' ]]; then cd dashboard/tools; fi
-  - if [[ $TRAVIS_BRANCH == 'master' ]]; then composer install; fi
-  - if [[ $TRAVIS_BRANCH == 'master' ]]; then php push-travis.php ../../tests/reports/phpunit.latest.xml; fi
+  - if [[ $TRAVIS_BRANCH == 'PAR-900' ]]; then cd devops/tools/php; fi
```
Comment (kalpaitch) [inline .travis.yml]: Can't merge this :)

File: Vagrantfile [hunk 2]
```diff
@@ -1,30 +1,20 @@
...
 sudo docker-compose up -d
 
 sh setup.sh
-
-date > /etc/vagrant_provision_end
 SCRIPT
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/xenial64"
+  config.vm.box = "netsensia/xenial64-docker"
```
Comment (stemount) [inline Vagrantfile]: private repo

File: Vagrantfile [hunk 3]
```diff
@@ -40,5 +30,6 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |v|
     v.memory = 3096
     v.cpus = 4
```
Comment (stemount) [inline Vagrantfile]: Let's change this to v.cpus = 1. Vagrant performs better as 1 CPU after many tests!

File: Vagrantfile [hunk 4]
```diff
@@ -1,30 +1,20 @@
...
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/xenial64"
+  config.vm.box = "netsensia/xenial64-docker"
+  config.vm.box_version = "1.0.0"
   config.vm.network "private_network", ip: "192.168.82.68"
   config.vm.network "forwarded_port", guest: 8111, host: 8111
   config.vm.network "forwarded_port", guest: 5411, host: 5411
 
+  config.ssh.username = "ubuntu"
+  config.ssh.password = "vagrant"
```
Comment (stemount) [inline Vagrantfile]: how come we want to force entering the user/pass?

messages image 2998082861

File: .travis.yml (hunk 1 above)
Comment (stemount) [reply .travis.yml]: +1

File: .travis.yml [hunk 5]
```diff
@@ -144,9 +144,9 @@ script:
...
+  - if [[ $TRAVIS_BRANCH == 'PAR-900' ]]; then composer install; fi
+  - if [[ $TRAVIS_BRANCH == 'PAR-900' ]]; then php push_test_results.php ../../../tests/reports/phpunit.latest.xml ../../../tests/reports/report.json; fi
```
Comment (stemount) [inline .travis.yml]: is this supposed to be `../../tests/reports/phpunit.latest.xml;`?

**Pull request closed without merging**